.circleci/config.yml

```diff
@@ -42,47 +42,58 @@ jobs:
             docker ps -a
 
       - run:
-          name: Run backend tests
+          name: Run database tests
```

Comment:
Did you notice that currently no tests are executed in the database and api tests? Also, the database sleep and manual start from a few lines down should probably be executed before all tests